container/container.go

```diff
@@ -234,10 +235,31 @@ func getHostname(taskInfo *mesos.TaskInfo) string {
 	return ""
 }
 
+// Map Task Env to Docker Env
+func AppendTaskEnv(envVars [] string, taskInfo * mesos.TaskInfo) [] string {
```

Comment:
I think you shouldn't put space between [] and `string` in Go. `go fmt` reformat it without space btw.